cibyl/orchestrator.py

```diff
@@ -152,3 +152,25 @@ def extend_parser(self, attributes, group_name='Environment',
                    hasattr(class_type, 'API'):
                     self.extend_parser(class_type.API, class_type.__name__,
                                        level=level+1)
+
+    def register_plugins(self, plugins):
+        """
+        Register plugins.
+        """
+        for plugin in plugins:
+            if hasattr(plugin, 'API_CLASS'):
+                for module, class_name in plugin.API_CLASS.items():
+                    loaded_module = __import__(f"{plugin.__name__}.{module}",
+                                               fromlist=[''])
+                    if hasattr(loaded_module, class_name):
```

Comment:
This call to `extend_parser` is not really necessary. If the plugin models are included in the Job API the call to `extend_parser` from main will correctly populate the parser

Reply:
I tried it without extend_parser. But I couldn't see plugin arguments. I think we need to keep it.

Reply:
The reason why you could not see them is that in the previous version it was not finding the Job's model and adding it to its API. If that happens, the call to extend parser in the main will add the plugin arguments